CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
 - Bump Sentry Gradle plugin to v2.1.5
 - Update translations: `am-ET`, `ta-LK`, `kn-IN`, `bn-BD`, `bn-IN`. `pa-IN`, `mr-IN`, `ta-IN`, `si-LK`, `sid-ET`, `te-IN`
 - Bump flipper to 0.134.0
+- Provide `AppUpdateHelpContact` based on the `Country`
+
+### Changes
+
+- [In Progress: 16 Feb 2021] Add stronger app update notifications
```

Comment:
We don't have notifications to update app yet, stronger does not make sense. Can we remove stronger and rename it to "Add app update reminders"

Reply:
We already show an app update dialog, we consider that to be a reminder or a notification. That's why we are adding a stronger "notification", not just system notification, but this whole system.

Reply:
It is not just notifying users, but majorly reminding them to update, just the `Add stronger app update reminders` makes more sense. WDYT?

Reply:
TBH I do think they are both same, but making the change and pushing it.